src/Workspaces/Core/Portable/FindSymbols/FindReferences/FindReferenceCache.cs

```diff
@@ -54,18 +55,27 @@ static async Task<FindReferenceCache> ComputeCacheAsync(Document document, Cance
     public readonly ISyntaxFactsService SyntaxFacts;
     public readonly SyntaxTreeIndex SyntaxTreeIndex;
 
+    /// <summary>
+    /// Not used by FAR directly.  But we compute and cache this while processing a document so that if we call any
+    /// other services that use this semantic model, that they don't end up recreating it.
+    /// </summary>
+#pragma warning disable IDE0052 // Remove unread private members
+    private readonly SemanticModel _nullableEnabledSemanticModel;
+#pragma warning restore IDE0052 // Remove unread private members
```

Comment:
this is part of the change.  ideally nothing will be using nullable-enabled-semantic-model (And i want to stamp out anything that is using it).  but if they do, this at least means we cache the results while processing the doc.

Reply:
next week i'll be setting BPs in the normal 'get semantic model' codepath so that i can update any final services still using it during FAR to stop.